models/stg_amazon_ads__ad_group_level_report.sql

```diff
@@ -23,40 +23,40 @@ final as (
     select 
         _fivetran_synced,
         ad_group_id,
-        attributed_sales_same_sku_14_d,
-        attributed_sales_same_sku_1_d,
-        attributed_sales_same_sku_30_d,
-        attributed_sales_same_sku_7_d,
+        attributed_sales_same_sku_14_d, 
+        attributed_sales_same_sku_1_d, 
+        attributed_sales_same_sku_30_d, 
+        attributed_sales_same_sku_7_d, 
         campaign_bidding_strategy,
-        click_through_rate,
+        click_through_rate, 
         clicks,
         cost,
-        cost_per_click,
-        date,
+        cost_per_click, 
+        date, 
```

Comment:
To be consistent with our other packages, can we rename these fields to be `date_day`.

Reply:
Please make this update across all staging models.

Reply:
done.